source/retryable-reads/retryable-reads.rst

```diff
@@ -0,0 +1,664 @@
+was made.
+
+2b. When to allow retry
+'''''''''''''''''''''''
+
+Drivers MUST only attempt to retry a read command if
+
+-  retryable reads are enabled **and**
+
+-  the selected server supports retryable reads **and**
+
+-  the first attempt yields a retryable error
+
+3. Deciding to allow retry, encountering the initial retryable error, and selecting a server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the driver decides to allow retry and the first attempt of a retryable read
+command encounters a retryable error, the driver MUST update its topology
+according to the Server Discovery and Monitoring spec (see `SDAM: Error Handling
+<https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst#error-handling>`__)
+and capture this original retryable error. Drivers should then proceed with
+selecting a server for the retry attempt.
+
+3a. Selecting the server for retry
+''''''''''''''''''''''''''''''''''
+
+If the driver cannot select a server for the retry attempt or the newly selected
+server does not support retryable reads, retrying is not possible and drivers
+MUST raise the original retryable error. In both cases, the caller is able to
+infer that an attempt was made.
+
+3b. Sending an equivalent command for the second attempt
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+After server selection, a driver MUST send a valid command to the newly selected
+server that is equivalent [2]_ to the initial command sent to the first
+server. If the driver determines that the newly selected server may not be able
+to support a command equivalent to the initial command, drivers MUST NOT retry
+and MUST raise the original retryable error
+
+The above requirement can be fulfilled in one of two ways:
+
+1. During the retry attempt, the driver SHOULD recreate the command while
+   adhering to that operation's specification's server/wire version
+   requirements. If an error occurs while recreating the command, then the
+   driver MUST raise the original retryable error.
+
+   a. For example, if the wire version dips from *W*\ :sub:`0` to *W*\ :sub:`1`
+      after server selection, and the spec for operation *O* notes that for wire
+      version *W*\ :sub:`1`, that field *F* should be omitted, then field *F*
+      should be omitted. If the spec for operation *O* requires the driver to
+      error out if field *F* is defined when talking to a server with wire
+      version *W*\ :sub:`1`, then the driver must error out and raise the
+      original retryable error.
+
+2. Alternatively, if a driver chooses not to recreate the command as described
+   above, then a driver MUST NOT retry if the server/wire version dips after
+   server selection and MUST raise the original retryable error.
+
+   b. For example, if the wire version dips after server selection, the driver
```

Comment:
Same comment as above (this should probably just be a second paragraph). Beyond that, I don't think it'd make sense to start with `b` here because it's the first sub-bullet.

Reply:
Oops, good catch, some formatting weirdness happened there. I've converted the bullet into a second paragraph per your suggestion :)